samples/matter/light_bulb/src/lighting_manager.h

```diff
@@ -27,7 +27,7 @@ class LightingManager {
 	void SetCallbacks(LightingCallback_fn aActionInitiated_CB, LightingCallback_fn aActionCompleted_CB);
 
 private:
-	static constexpr uint8_t kMaxLevel = 255;
+	static constexpr uint8_t kMaxLevel = 254;
```

Comment:
Do you think it would be possible to somehow consider MinLevel and MaxLevel attribute values from LevelControl cluster? As far as I understand currently we are able to increment level value and go outside of this range, right?

Reply:
Those attributes are read-only and currently set to 0-254, so it's impossible to go outside this range, but I can take this range from the cluster instead of using the constants.

Reply:
I meant that our code should be consistent with cluster configuration and work even if someone will regenerate the files and change the defaults

Reply:
Ok, changed the code.